### ISSUE TITLE
Incorrect handling of disable-hostname-verification and allow-self-signed-certificates

### ISSUE THREAD

**Issue by ngrigoriev** [JanusGraph/janusgraph#731]
Our internal code review has revealed a problem with the PR for #645:

```
            if (config.has(ElasticSearchIndex.SSL_DISABLE_HOSTNAME_VERIFICATION) &&
                    config.has(ElasticSearchIndex.SSL_DISABLE_HOSTNAME_VERIFICATION)) {
                ...

            if (config.has(ElasticSearchIndex.SSL_ALLOW_SELF_SIGNED_CERTIFICATES) &&
                    config.has(ElasticSearchIndex.SSL_ALLOW_SELF_SIGNED_CERTIFICATES)) {
                ...
```

Should have been "get" in the second place. I will add that to the tests and fix it now.

**Comment by sjudeng** [comment]
Thanks for catching and submitting fix!

**Comment by mbrukman** [comment]
This will be fixed by PR #732.
